assets/src/components/cd/services/ServiceDependenciesColumns.tsx

```diff
@@ -0,0 +1,16 @@
+import { ServiceDependency } from 'generated/graphql'
+import { createColumnHelper } from '@tanstack/react-table'
+
+const columnHelper = createColumnHelper<Nullable<ServiceDependency>>()
+
+const ColName = columnHelper.accessor('name', {
+  id: 'name',
+  header: 'Name',
+})
+
+const ColStatus = columnHelper.accessor('status', {
```

Comment:
I think we should render the status as a chip with appropriate color.  Also, it's realistically only set as either nil or `HEALTHY` at the moment so there'd need to be ternary logic like `color={status === 'HEALTHY' ? 'success' : 'warn'}`

Reply:
updated